cmd/juju/model/configcommand.go

```diff
@@ -155,7 +221,10 @@ func (c *configCommand) handleZeroArgs() error {
 func (c *configCommand) handleOneArg(arg string) error {
 	// We may have a single config.yaml file
 	_, err := os.Stat(arg)
-	if err == nil || strings.Contains(arg, "=") {
+	if err == nil {
+		// Handle yaml files
+		return c.parseYAMLFile(arg)
```

Comment:
This makes reading the code easier to follow, as previously it just jumped into `parseSetKeys` and it was expected you knew that `setOptions.Set()` handled files.